src/configuration/routes/cache.md

```diff
@@ -25,56 +23,64 @@ https://{default}/:
 
 In this example, requests will be cached based on the URI, the `Accept` header, `Accept-Language` header, and `X-Language-Locale` header; Any response that lacks a `Cache-Control` header will be cached for 60 seconds; and the presence of any cookie in the response will disable caching of that response.
```

Comment:
This should also note that any header listed in Vary will be part of the cache key, too.